websites/B/bilibilitv/presence.ts

```diff
@@ -45,8 +45,8 @@ presence.on("UpdateData", async () => {
 				?.getAttribute("poster")
 				?.split("@")[0] ||
 			document
-				.querySelector<HTMLMetaElement>('meta[name="og:image"]')
-				?.content?.split("@")[0];
+				.querySelector<HTMLMetaElement>('meta[property="og:image"]')
+				.content.split("?")[0];
```

Comment:
This is so it falls back to the logo when the element doesn't exist.
```suggestion
				?.content?.split("?")?.[0] ?? "https://cdn.rcd.gg/PreMiD/websites/B/bilibilitv/assets/logo.png";
```